packages/server/lib/modes/run.ts

```diff
@@ -321,9 +321,9 @@ async function postProcessRecording (options: { quiet: boolean, videoCompression
 
   // once this ended promises resolves
   // then begin processing the file
-  // dont process anything if videoCompress is off
+  // don't process anything if videoCompress is off
   // or we've been told not to upload the video
-  if (options.videoCompression === false || options.shouldUploadVideo === false) {
+  if (options.videoCompression === false || options.videoCompression === 0 || options.shouldUploadVideo === false) {
```

Comment:
This is the bug fix.

Reply:
`if (!options.videoCompress || !options.shouldUploadVideo)` ?

Reply:
i'd rather be explicit